Doc/library/pickle.rst

```diff
@@ -278,15 +278,15 @@ The :mod:`pickle` module defines three exceptions:
 .. exception:: PicklingError
 
    Error raised when an unpicklable object is encountered by :class:`Pickler`.
-   It inherits :exc:`PickleError`.
+   It inherits from :exc:`PickleError`.
```

Comment:
Can you also change the one on line 275 just above here? (A little harder to spot since it's broken over two lines.)

Reply:
Got it.